### ISSUE TITLE
ActivityResultLauncher Integration

### ISSUE THREAD

**Issue by omkar-tenkale** [anggrayudi/SimpleStorage#17]
The current approach involves using onActivityResult()

But it's deprecated now
https://stackoverflow.com/questions/62671106/onactivityresult-method-is-deprecated-what-is-the-alternative

Can we get the SimpleStorage intent, eg 

 ```
ActivityResultLauncher<Intent> launchSomeActivity = registerForActivityResult(
     new ActivityResultContracts.StartActivityForResult(),
     new ActivityResultCallback<ActivityResult>() {
              @Override
              public void onActivityResult(ActivityResult result) {
                 storage.onActivityResult(result);
               }
      });

    public void openYourActivity() {
            Intent intent =storage.getFilePickerIntent();
            launchSomeActivity.launch(intent);
      }
```

**Comment by shashifreeze** [comment]
@omkar-tenkale you are right.

**Comment by anggrayudi** [comment]
Version `0.6.0-SNAPSHOT` is released from branch `release/0.6.0`. Please test it whether this meets your expectation.

**Comment by omkar-tenkale** [comment]
HI @anggrayudi 

Thanks for the effortw

But sadly I no longer use the lib because I couldn't create a release build with this lib in my app's dependencies

I had to remove all the code and use SAF methods with hacks

Debug APK was generated but release apk generation would fail with errors related to kotlin and java exceptions i don't remember well now

My app didn't had any kotlin code and as this lib depends on many complex package that might have caused some troubles

Anyways, Thanks for your effort.I hope others will benefit if not me :)
